tests/mongoPersistanceService.test.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require('mongoose')
+const SensorFault = require('../models/SensortFault')
+const {saveFaultySensor} = require("../services/mongoPersistanceService")
+
+beforeAll(async () => {
+    const full_url = process.env.MONGO_URL.split('/').slice(0, -1).join('/') + '/' + global.__MONGO_DB_NAME__ 
```

Comment:
I believe it would be better to keep the url inside a .env file.

Reply:
@michaelbahchevanov  I think you meant the other file, right? Will do